svg.js.d.ts

```diff
@@ -754,10 +754,10 @@ declare namespace svgjs {
     // selector.js
     interface Library {
         get(id: string): Element;
-        select(query: string, parent: HTMLElement): Set;
+        select(query: string): Set;

```

Comment:
`SVG.select()` has the method signature `select(query, parentNode)` so removing the second parameter will most likely not work. But it is optional so adding a simple `?` might help instead

Reply:
I'm not the typescript master either (yet) ;) and it seems that you are right. The `?` will do the job, so the whole line should look like this: `select(query: string, parent?: HTMLElement): Set;'. So basically to make it works, it's enough to make this parameter as optional as it stands in the documentation.